bindings/python/iota_sdk/types/node_info.py

```diff
@@ -217,12 +213,10 @@ class ProtocolParameters:
     staking_unbonding_period: str
     validation_blocks_per_slot: int
     punishment_epochs: str
```

Comment:
EpochIndex

Reply:
yeah I was confused about this too

Reply:
It's just an amount of epochs, not an epoch index

Reply:
In Rust we also don't use EpochIndex there

Reply:
fair enough

Reply:
Kind of the same for committable age etc tbh. I actually think we should change it in Rust as well then? It's not an index sure, but it's a delta index, maybe it's interesting to make sure they are the same types because we end up doing operations on them. Idk 🤷🏻‍♂️

Reply:
Would rather also remove the SlotIndex there, computation should be possible wit the inner value

Reply:
As you prefer, as long as we're consistent